model/schema/ingest.yaml

```diff
@@ -260,6 +268,7 @@ classes:
       - allele_origin_association_ingest_set
       - agm_ingest_set
       - sqtr_ingest_set
+      - ontology_closure_ingest_set
```

Comment:
@chris-grove @sierra-moxon not sure we need an ingest for this considering the data is coming directly from owl files.